src/settings/VoucherNumber/VoucherNumber.css

```diff
@@ -0,0 +1,6 @@
+@import "@folio/stripes-components/lib/variables.css";
+
+.allowNumberEdit {
```

Comment:
What the reason of these styles?

Reply:
This style need for draw this line 
 
![image](https://user-images.githubusercontent.com/4868618/63165862-4aaa8680-c035-11e9-8649-12da1cf8bde3.png)